CHANGELOG.rst

```diff
@@ -2,13 +2,46 @@
 Changes
 =======
 
-1.7.0 (2024-??-??)
+1.7.0 (2024-08-26)
 ------------------
 
 **New features**
 
-* Made user emails unique to prevent two users logging in with the same email, causing an error
+* [#169] Made user emails unique to prevent two users logging in with the same email, causing an error
+* [#151] Added 2FA to the Admin
+* [#157] Optimized deleting abonnement with a lot of notifications in the Admin
```

Comment:
Guess we could also add this:

* [maykinmedia/open-api-framework#21] display version in admin